pretext/project/refactor.py

```diff
@@ -38,6 +38,11 @@ class Format(str, Enum):
     CUSTOM = "custom"
 
 
+class Publication(pxml.BaseXmlModel, tag="publication", search_mode="unordered"):
```

Comment:
`PublicationDirectories`?

Reply:
or `ManagedDirectories`? I don't want someone to see this class and think it's validating the whole publication file.